bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-toolbar/component.jsx

```diff
@@ -71,13 +71,20 @@ class WhiteboardToolbar extends Component {
   constructor(props) {
     super(props);
 
-    const { annotations } = this.props;
+    const { annotations, multiUser } = this.props;
 
     let annotationSelected = {
       icon: 'hand',
       value: 'hand',
     };
 
+    if (multiUser) {
```

Comment:
Can't it being made using a ternary?

Reply:
The default tool is 'hand':
```
let annotationSelected = {
      icon: 'hand',
      value: 'hand',
    };
```
So if the multi user enabled the default tool must be the 'pen_tool' then:
```
if (multiUser) {
      annotationSelected = {
        icon: 'pen_tool',
        value: 'pencil',
      };
    }
```
But, in case of the 'hand', and 'pen_tool' disabled in multiUserTools settings, the first available tool must be selected, then we have another if to set the `annotationSelected`, like this:
```
if (!annotations.some(el => el.value === annotationSelected.value) && annotations.length > 0) {
      annotationSelected = annotations[annotations.length - 1];
    }
```

Reply:
I recall an issue that came up previously, where having the pencil as the default was causing users to unintentionally mark up the whiteboard. Do we still want to default to the 'pen_tool'?

Reply:
Afaik the unintentional marks on the whiteboard were in normal mode, i.e. non multi-user.

Reply:
Yeah the issue was for the normal mode, was wondering if it would apply to the multi-user

Reply:
For multi-user whiteboard mode the intent is to create marks from the get-go, so I think all is fine. Still, thanks for reminding us about that previous use case!

Reply:
Is the intention to always immediately start writing? It's always the intention with us when we test it because we want to stress it, but is it always the case for others?

Reply:
Having this check in the constructor is also kind of strange because now you'd have different paths if a person gets presenter with or without multi-user on.